launcher/ui/themes/BrightTheme.cpp

```diff
@@ -18,19 +18,19 @@ bool BrightTheme::hasColorScheme()
 QPalette BrightTheme::colorScheme()
 {
     QPalette brightPalette;
-    brightPalette.setColor(QPalette::Window, QColor(239,240,241));
-    brightPalette.setColor(QPalette::WindowText, QColor(49,54,59));
-    brightPalette.setColor(QPalette::Base, QColor(252,252,252));
-    brightPalette.setColor(QPalette::AlternateBase, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ToolTipBase, QColor(49,54,59));
-    brightPalette.setColor(QPalette::ToolTipText, QColor(239,240,241));
-    brightPalette.setColor(QPalette::Text,  QColor(49,54,59));
-    brightPalette.setColor(QPalette::Button, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ButtonText, QColor(49,54,59));
+    brightPalette.setColor(QPalette::Window, QColor(255,255,255));
+    brightPalette.setColor(QPalette::WindowText, QColor(17,17,17));
+    brightPalette.setColor(QPalette::Base, QColor(250,250,250));
+    brightPalette.setColor(QPalette::AlternateBase, QColor(246,246,246));
+    brightPalette.setColor(QPalette::ToolTipBase, QColor(17,17,17));
+    brightPalette.setColor(QPalette::ToolTipText, QColor(255,255,255));
+    brightPalette.setColor(QPalette::Text,  Qt::black);
+    brightPalette.setColor(QPalette::Button, QColor(249,249,249));
+    brightPalette.setColor(QPalette::ButtonText, Qt::black));
```

Comment:
as the CI builds state, you forgot a bracket there
```suggestion
    brightPalette.setColor(QPalette::ButtonText, Qt::black);
```